mava/networks.py

```diff
@@ -80,13 +84,13 @@ class DiscreteActionHead(nn.Module):
     action_dim: int
 
     @nn.compact
-    def __call__(self, x: chex.Array, observation: Observation) -> distrax.Categorical:
+    def __call__(self, x: chex.Array, action_mask: chex.Array) -> distrax.Categorical:
```

Comment:
As discussed offline: 

```suggestion
    def __call__(self, obs_embedding: chex.Array, observation: Observation) -> distrax.Categorical:
```

Reply:
Just added a bit more detail than your suggestion - but very much along the same lines